src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js

```diff
@@ -5,10 +5,21 @@ const testConfig = require("../../../config");
 module.exports = async function () {
 
     const I = this;
+    //Before Letters Page Starts
+    I.waitForText('Letters', testConfig.TestTimeToWaitForText);
+    I.see('Case Number:')
+    I.see('List of correspondence items');
+    I.see('Top Level');
     I.waitForText(commonConfig.lettersCorrespondence, testConfig.TestTimeToWaitForText);
     I.selectOption('#correspondenceType_topLevel_Documents', commonConfig.lettersCorrespondence);
     I.waitForText(commonConfig.lettersCorrespondence1, testConfig.TestTimeToWaitForText);
     I.selectOption('#correspondenceType_part_2_Documents', commonConfig.lettersCorrespondence1);
+    I.see('Letters');
     I.click(commonConfig.continue);
     I.click(commonConfig.submit)
+    //After submitting letters
+    I.see('Please download the document from : ')
```

Comment:
please add a wait statement here. 
after clicking submit button the next page could take some time to load

Reply:
before the `I see` step

Reply:
Added a wait statement.

Reply:
Yes, waited for a header, and then saw the statement